eng/versioning/external_dependencies.txt

```diff
@@ -60,15 +60,20 @@ org.powermock:powermock-api-mockito2;2.0.2
 org.powermock:powermock-module-junit4;2.0.2
 org.slf4j:slf4j-api;1.7.28
 org.slf4j:slf4j-simple;1.7.25
-org.springframework.boot:spring-boot-actuator-autoconfigure;2.2.0.RELEASE
-org.springframework.boot:spring-boot-autoconfigure;2.2.0.RELEASE
-org.springframework.boot:spring-boot-autoconfigure-processor;2.2.0.RELEASE
-org.springframework.boot:spring-boot-configuration-processor;2.2.0.RELEASE
-org.springframework.boot:spring-boot-dependencies;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-test;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-validation;2.2.0.RELEASE
-org.springframework.boot:spring-boot-starter-web;2.2.0.RELEASE
+
+## Spring boot dependency versions
+org.springframework.boot:spring-boot-actuator-autoconfigure;2.3.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure-processor;2.3.0.RELEASE
+org.springframework.boot:spring-boot-autoconfigure;2.3.0.RELEASE
+org.springframework.boot:spring-boot-configuration-processor;2.3.0.RELEASE
+org.springframework.boot:spring-boot-dependencies;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-test;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-validation;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-web;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter;2.3.0.RELEASE
+org.springframework.boot:spring-boot;2.3.0.RELEASE
+org.springframework.boot:spring-boot-starter-parent;2.3.0.RELEASE
+
 org.springframework.security:spring-security-config;5.2.0.RELEASE
```

Comment:
Do we need to upgrade these spring security versions too?

Reply:
Updated.